module/questions/helper.js

```diff
@@ -80,7 +80,7 @@ module.exports = class QuestionsHelper {
           allValues["visibleIf"] = new Array;
           allValues["question"] = new Array;
 
-          let evidenceMethod = parsedQuestion["evidenceMethod"];
+          let evidenceMethod = parsedQuestion["evidenceMethod"] ? parsedQuestion["evidenceMethod"] : Object.keys(evidenceCollectionMethodObject)[0];
```

Comment:
@deepakr123 - Changes to be done to use the same function.

Reply:
resolved